remote-api/src/test/java/org/alfresco/rest/api/impl/rules/RuleSetsImplTest.java

```diff
@@ -172,6 +173,32 @@ public void testLinkingToRuleSet()
         then(nodeServiceMock).shouldHaveNoMoreInteractions();
     }
 
+    @Test
+    public void testLinkToRuleSet()
+    {
+        NodeRef childNodeRef = new NodeRef(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, "dummy-child-id");
+
+        given(nodeValidatorMock.isRuleSetNode(any())).willReturn(true);
+        given(ruleServiceMock.hasRules(any(NodeRef.class))).willReturn(true, false);
+        given(runtimeRuleServiceMock.getSavedRuleFolderAssoc(any(NodeRef.class))).willReturn(assocRef);
+        given(assocRef.getChildRef()).willReturn(childNodeRef);
+
+        //when
+        assertEquals(ruleSets.linkToRuleSet(FOLDER_ID,LINK_TO_NODE_ID).getId(), childNodeRef.getId());

```

Comment:
Assuming that test methods should be divided into 3 blocks: given, when and then, assertions should be placed in the last block. At first glance I didn't noticed that here is an assertion.